src/template/src/Home.jsx

```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
           </Link>
           <h1> {project.name} </h1>
           <p>
-            {{i18n_gettingStarted}}
+            {{ i18n_gettingStarted }}
```

Comment:
This space change is something done by your linter, @Mortaro?
Last time it simply broke the create, pay attention or it going to do it again